projects/storefrontlib/src/shared/components/checktable/checktable.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, Input, Output } from '@angular/core';
+import { EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'cx-checktable',
+  templateUrl: './checktable.component.html',
+})
+export class ChecktableComponent {
+  @Input()
+  columns: Array<{
+    key: string;
+    value: string;
+    cxRoute?: string;
+    checked?: boolean;
+  }>;
+
+  @Input()
+  tableData: Array<any>;
+
+  @Output() toggle = new EventEmitter<any>();
+
+  selectedValues: Array<any>;
+
+  onToggle(event, row) {
+    this.toggle.emit({ value: event.target.value, row });
```

Comment:
sorry, I forgot that for checkbox we have event.target.checked

Reply:
fixed now